contracts/JobRegistry.sol

```diff
@@ -172,6 +214,36 @@ contract JobRegistry {
         );
     }
 
+
+    /**
+     * @notice Allows the employer to assign an employee to the job
+     * @param _jobId Job identifier
+     */
+    
+    function assignEmployeeToJob(uint256 _jobId, uint256 _employeeId) public view {
+
+        OpenJob storage openjob = openJobs[_jobId];
+        uint256 senderId = tlId.walletOfOwner(msg.sender);
+        
```

Comment:
A rajouter:
       require(
            _employeeId != _employerId,
            "Employee and employer can't be the same"
        );

Reply:
C'est ajouté